docs/USAGE.md

```diff
@@ -111,6 +111,37 @@ You can find the scss files here:
 - dist/styles/variables - containing all the variables for colors, typography, etc
 - dist/lib - containing all the components styles
 
+### Canopy Fonts
+
+Canopy consists of two fonts; Roboto and Lyon. Roboto is included in the distributed Canopy package, and will load automatically when you have installed Canopy correctly.
+
+Due to our commericial license agreement, Lyon is not included in the Canopy distributed package, so you need to include it in the **assets** directory of your application. Canopy still references Lyon in the \`\`@font-face\`\` rulesets, so you don't need to implement them, but you do need to ensure that the path to the font in your assets directory is the same as the paths specified by Canopy in \`\`@font-face\`\`.
+
+Canopy's path for Lyon is **/assets/fonts/lyon/**, therefore you need to put the font files in the following location:
```

Comment:
I think there is a line to add here along the lines of...

> Please ensure you have the correct licensing agreement in place before using the Lyon font in your application.

I would like to be able to link that text to somewhere, but not sure where.

Reply:
Done ✅